.github/workflows/ci.yml

```diff
@@ -0,0 +1,100 @@
+name: CI
+
+on: 
+  push:
+
+
+jobs:
+
+  activate:
+    runs-on: ubuntu-latest
+    if: |
+      github.repository == 'feelpp/IfcOpenShell' &&
```

Comment:
should this hardcoded fork repo be in here?

Reply:
nope of course not